deck.go

```diff
@@ -70,6 +72,42 @@ func LoadDeck(dev *streamdeck.Device, base string, deck string) (*Deck, error) {
 		d.Widgets = append(d.Widgets, w)
 	}
 
+	watcher, err := fsnotify.NewWatcher()
+	if err == nil {
+		err = watcher.Add(path)
+		if err == nil {
+
+			go func() {
+				for {
+					select {
+					case event := <-watcher.Events:
+						if currentDeck == path {
+							fmt.Printf("Change:  %s: %s\n", event.Op, event.Name)
+							d, err := LoadDeck(dev, base, deckName)
+							if err != nil {
+								fatal(err)
+							}
+							err = dev.Clear()
+							if err != nil {
+								fatal(err)
+							}
+
+							deck = d
+							deck.updateWidgets()
+							return
+						}
+					case error := <-watcher.Errors:
+						fmt.Printf("Watcher had an error: %s\n", error)
```

Comment:
I know I've just introduced that in a commit a few minutes ago, but we should try to write all errors to `os.Stderr`:

```go
fmt.Fprintf(os.Stderr, ....)
```